services/CollectionService/.gitignore

```diff
@@ -1 +1,3 @@
 vendor/
+composer.lock
+composer.phar
```

Comment:
From personal experience, including the lock files always ends with some sort of confusion over why things won't update when people are testing changes.

I understand this is contrary to what stack overflow says.

It's bitten me before many many times, and I make it a habit not to.  With the understanding that I need to know what I'm doing if deploying straight from the git repo on a production environment.